pages/osx/g[.md

```diff
@@ -0,0 +1,7 @@
+# g[
+
+> This command is an alias of GNU `[`.
+
+- View documentation for the original command:
+
+`tldr --platform linux [`
```

Comment:
the `--platform` is not required in the spec, we should use `-p` like the `tldr` page has
```suggestion
`tldr -p linux [`
```

Reply:
I'm sorry, but who came up with that idea?! For a project that emphasizes long and readable arguments as much as tldr does, that is one of the dumbest things I've heard in a long time.

Reply:
I wasn't here in that time. @waldyrious wdyt?

Reply:
Now, I am questioning myself 😄 , Should we update all the alias pages too then?

Reply:
> I'm sorry, but who came up with that idea?! For a project that emphasizes long and readable arguments as much as tldr does, that is one of the dumbest things I've heard in a long time.

Until project doesn't have formal rule about using long options it's up to you (here) whether to use them or not.